person.rb

```diff
@@ -8,13 +9,14 @@ def initialize(age, name = 'Unknown', parent_permission = DEFAULT_BOOL)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
 
     @corrector = Corrector.new
   end
 
   attr_accessor :name, :age
 
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] for `@rentals` you might want to set up both the setter and the getter as well.